### ISSUE TITLE
OK Stuff and such...

### ISSUE THREAD

**Issue by vadis365** [ganymedes01/TheErebus#226]
This Sunday is earmarked for release day...lol
I still have to make the dungeon less rare and get someone to *fudging test the boss.
I have a plan for tonight to make that better...
Andre and myself have fixed the obvious errors that existed with regards to oregen and recipes and such. Andre has also added the bone block grave thingy :) . Cricket has made good progress with cooking bits (I guess these will remain WIP for the next release and can be hidden from creative menu and NEI easily) and added some achievements . Dylan has made a few textures for the doors which are kind of half done....grrrr....named wrong and no item icons for them too...grrrr.
So, in summary, some progress but still a lot of stuff to make right before Sunday.

Dylan will have to do all the admin stuff - like posting links and *shizz - of which he has had plenty of time to prepare himself for (no excuses!). Andre will be away on his travels quite soon, so, Dylan please do the stuff because I won't.

So,  lets slap on a fake smile and plough through this *ship one more time...

Me.

*edited to be Jade friendly (Andre knows what I mean )
